eval/bedrock_runner.py

```diff
@@ -0,0 +1,202 @@
+import boto3
+import json
+import os
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from typing import Optional
+
+from eval.eval import compare_query_results
+import pandas as pd
+from utils.gen_prompt import generate_prompt
+from utils.questions import prepare_questions_df
+from utils.creds import db_creds_all
+from tqdm import tqdm
+from time import time
+from utils.reporting import upload_results
+
+bedrock = boto3.client(service_name="bedrock-runtime")
+
+
+def process_row(row, model_id, decimal_points):
+    start_time = time()
+
+    body = json.dumps(
+        {
+            "prompt": row["prompt"],
+            "max_gen_len": 400,
+            "temperature": 0,
+            "top_p": 1,
+        }
+    )
```

Comment:
curious where are the parameters for their api documented? I looked into [this](https://docs.aws.amazon.com/bedrock/latest/APIReference/CommonParameters.html) but to little avail

Reply:
I couldn't really find good documentation, but the bedrock page [here](https://us-west-2.console.aws.amazon.com/bedrock/home?region=us-west-2#/providers?model=meta.llama3-70b-instruct-v1:0) was helpful (tiny usage example at the very bottom)!